### PR TITLE
JBIDE-27652: Upgrade openshift/kubernetes client

### PR DESCRIPTION
**Pull request by jeffmaury**
Signed-off-by: Jeff MAURY <jmaury@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

File: plugins/org.jboss.tools.openshift.client/.classpath [hunk 1]
```diff
@@ -19,11 +19,11 @@
...
-	<classpathentry exported="true" kind="lib" path="lib/openshift-client-4.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/servicecatalog-client-4.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/servicecatalog-model-4.9.1.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/jackson-datatype-jsr310-2.10.3.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/kubernetes-client-4.11.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/kubernetes-model-core-4.11.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/openshift-client-4.11.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/servicecatalog-client-4.11.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/servicecatalog-model-4.11.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/openshift-model-4.11.2.jar"/>
```
Comment (sbouchet) [inline plugins/org.jboss.tools.openshift.client/.classpath:27]: is that intentional to have removed jackson-datatype drom classpath ? it's still referenced in the Manifest.mf

Comment (jeffmaury) [reply plugins/org.jboss.tools.openshift.client/.classpath:28]: My bad, good catch

File: plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/odo/utils/ConfigHelper.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
 import io.fabric8.kubernetes.api.model.Config;
 import io.fabric8.kubernetes.api.model.ConfigBuilder;
 import io.fabric8.kubernetes.api.model.Context;
```
Comment (sbouchet) [inline plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/odo/utils/ConfigHelper.java:17]: please remove unused import.

**Review by sbouchet — APPROVED** [review]
LGTM

**Pull request merged**